src/djimatrice_drone.py

```diff
@@ -0,0 +1,235 @@
+import roslibpy
+import numpy as np
+from drone import Drone
+from enum import Enum
+
+class DjiMatriceDrone(Drone):
+    '''
+    DJI Matrice API translator layer. 
+    Please see https://github.com/dji-sdk/Onboard-SDK-ROS for more information about the DJI SDK.
+    To use this, please install [TODO] on the DJI drone's onboard computer.
+    '''
+
+    drone_type = "DjiMatrice"
+
+    class DroneTaskControl(Enum):
+        GO_HOME = 1
```

Comment:
If possible, we should add a single-line comment explaining why the numbers for this Enum aren't just 1,2,3,4,etc. instead of 1, 4, and 6.